ontology/yaml/resources/HVAC/entity_types/PMP.yaml

```diff
@@ -43,6 +43,77 @@ PMP_SS_VSC_WDPM:
   - VSC
   - WDPM
 
+PMP_SS_WDT:
+  description: "Variable speed pump with temperature reading between return/supply water"
+  is_canonical: true
+  implements:
+  - PMP
+  - SS
+  - WDT
+  opt_uses:
+  - schedule_run_command
+  - control_mode
+  - run_mode
+  - lost_power_alarm
+  - control_status
+
+PMP_SSALS:
```

Comment:
Perhaps SS needs to be updated with these fields which you are re-using everywhere. Then no new canonical types are needed.

Reply:
This was wrongly defined on pump, this has been moved to HWS and renamed as HWS_SS_ALS